compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/expressions/DynamicParamNode.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.ballerinalang.model.tree.expressions;
+
+/**
+ * Represent named arguments.
```

Comment:
We can probably include more details, right? This is not quite a named arg?

Reply:
I added in `BLangDynamicParamExpr` class(Which extend from this). Do I need to add the same thing here?

Reply:
Yes, we need to sufficiently describe this here. "Represent named arguments." kinda indicates that it represents all named arguments which is not the case.

Reply:
Changed.